pkg/manifestgen/manifestgen.go

```diff
@@ -44,10 +44,19 @@ type Options struct {
 	// content can be read
 	SBOMWriter SBOMWriterFunc
 
+	// WarningsOutput will recieve any warnings that are part
+	// of the manifest generation. If that options is empty
```

Comment:
```suggestion
	// of the manifest generation. If it is unset
```
???

Reply:
I like this suggestion, feels less clunky than my original